crates/cxx-qt-gen/src/writer/rust/mod.rs

```diff
@@ -47,6 +48,15 @@ pub fn write_rust(generated: &GeneratedRustBlocks, include_path: Option<&str>) -
         cxx_qt_mod_contents.extend_from_slice(&fragment.cxx_qt_mod_contents);
     }
 
+    let docs: Vec<Attribute> = cxx_mod
+        .attrs
+        .iter()
+        .filter(|attr| path_compare_str(attr.meta.path(), &["doc"]))
+        .cloned()
+        .collect();
+
+    cxx_mod.attrs = vec![]; // Reset all attributes to stop cxx_qt::bridge being passed through
+
```

Comment:
instead see the comment further back about how we maybe should store the module as items and then rebuild here :thinking:

Reply:
Done with use of new PassthroughMod struct